conda_libmamba_solver/solver.py

```diff
@@ -1004,8 +1004,17 @@ def _notify_conda_outdated(
             # This check can be silenced with a specific option in the context or in quiet mode
             return
 
-        current_conda_prefix_rec = PrefixData(context.conda_prefix).get("conda", None)
-        if not current_conda_prefix_rec:
+        # manually check base prefix since `PrefixData(...).get("conda", None) is expensive
+        # once prefix data is lazy this might be a different situation
+        current_conda_prefix_rec = None
+        conda_meta_prefix_directory = os.path.join(context.conda_prefix, "conda-meta")
+        if os.path.lexist(conda_meta_prefix_directory):
```

Comment:
```suggestion
        if os.path.lexists(conda_meta_prefix_directory):
```

Typo? And why `lexists` instead of `exists`?

Reply:
Yeup fixed!